terminado/websocket.py

```diff
@@ -48,7 +48,14 @@ def _cast_unicode(s):
 
 class TermSocket(tornado.websocket.WebSocketHandler):
     """Handler for a terminal websocket"""
-    def initialize(self, term_manager):
+    def initialize(self, term_manager, keep_alive_time=5):
```

Comment:
I think I'd prefer that this be off by default, so it's not adding any complexity for people who don't need it.